dangerzone/isolation_provider/qubes.py

```diff
@@ -130,7 +130,7 @@ def is_qubes_native_conversion() -> bool:
         # This disambiguates if it is running a Qubes targetted build or not
         # (Qubes-specific builds don't ship the container image)
 
-        compressed_container_path = get_resource_path("container.tar.gz")
+        compressed_container_path = get_resource_path("container.tar")
```

Comment:
I guess we can also rename `compressed_container_path` to `container_tarball`

Reply:
Done in 5f4f82a